Positioning2/style.css

```
@@ -0,0 +1,66 @@
+.parent {
+    background-color: rgb(230, 10, 10)7;
+    display: flex;
+    width: 555px;
+    height: 191px;
+
+    border-style: dotted;
+    border-color: rgb(87, 67, 93);
+    border-radius: 10px;
+    border-width: 5px;
+
+    top: 75px;
+
+    text-align: center;
+}
+
+.parent {
+    background-color: rgb(168, 176, 167);
+    top: 0;
+    left: 50%;
+}
```

Feedback:

- mai indicat ar fi sa avem declarata 1 singura data clasa .parent pentru a fi mai usor de citit
- positionul pentru parinte trebuie setat `relative` pentru a putea misca copii cu `absolute`


```suggestion
.parent {
      background-color: rgb(168, 176, 167);
      position: relative;
      width: 500px;
      height: 191px;
      border-style: dotted;
      border-color: rgb(87, 67, 93);
      border-radius: 10px;
      border-width: 5px;
      text-align: center;
}
```